### PR TITLE
Add a code of conduct

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [ ] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [ ] HISTORY.rst has been updated (with summary of main changes)
  - [ ] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added

### What kind of change does this PR introduce?

* Adds a code of conduct document (Contributor Covenant), same as the one used for xclim

### Does this PR introduce a breaking change?

No.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
```
Comment (juliettelavoie) [inline CODE_OF_CONDUCT.md:55]: un adresse email xscen ?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+support@ouranos.ca.
```
Comment (juliettelavoie) [inline CODE_OF_CONDUCT.md:63]: Qui s'occupe de cette adresse?

**Review by juliettelavoie — APPROVED** [review]
☮

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (Zeitsperre) [reply CODE_OF_CONDUCT.md:63]: C'est une adresse utilisée pour répondre aux questions. Présentement, ce n'est pas monitoré, mais je dois occuper de ça vu que deux projets demandent ça. C'est dans mon plan de mieux servir la communauté GitHub.

**Review by RondeauG — APPROVED** [review]

**Pull request merged**